README.md

```diff
@@ -1,27 +1,39 @@
 # ssb-db
 
-[secret-stack](https://github.com/ssbc/secret-stack) plugin which provides storing of valid secure-scuttlebutt messages in an append-only log.
+[secret-stack](https://github.com/ssbc/secret-stack) plugin which provides storing of valid secure-scuttlebutt 
+messages in an append-only log.
 
-## What does it do?
+## Table of contents
 
-`ssb-db` provides tools for dealing with unforgeable append-only message 
-feeds. You can create a feed, post messages to that feed, verify a feed created by
-someone else, stream messages to and from feeds, and more (see [API](#api)).
+[What does it do?](#what-does-it-do) | 
+[Example](#example) | 
+[Concepts](#concepts) | 
+[API](#api) | 
+[Stability](#stability) | 
+[License](#License) | 
```

Comment:
Did you intend for these to appear on separate lines? if so, they're still not. MD is weird sometimes

Reply:
Since there are no double line breaks, they are rendered as one. It's just to make it more readable in the terminal or in text editors that don't break the lines.

Reply:
ah right, nice